website/src/pages/case-studies/sofi-moves-fast-on-hundreds-of-microservices.tsx

```diff
@@ -0,0 +1,66 @@
+                image: '/case-studies/ursula-robertson-sofi.jpg',
+            }}
+            // pdf="https://storage.googleapis.com/sourcegraph-assets/SoFi-Sourcegraph-case-study.pdf"
+        >
+            <ContentSection color="white" className="col-md-6">
+                <div className="container">
+                    <p>
+                        SoFi helps its over 900,000 members achieve financial independence to realize their ambitions
+                        with products for borrowing, saving, spending, investing, and protecting their money. With
+                        Sourcegraph, SoFi can innovate and move quickly while keeping up with hundreds of microservices.
+                    </p>
+                    <h2 className="pt-5 pb-1">The need for cross-repository code search</h2>
+                    <p>
+                        When SoFi (Social Finance Inc.) decided to switch their codehost from Bitbucket to GitLab
+                        earlier this year, they quickly realized they would need a more powerful code search tool to
+                        search over their hundreds of repositories. They also saw an increase in issues due to
+                        overlooked code interdependencies when newer engineers committed their code changes. While
+                        GitLab fulfilled many of their code host needs, it didn’t offer a good way to investigate the
+                        codebase across projects. The need for fast and accurate code search and code intelligence led
+                        the engineering team to install Sourcegraph. As a FinTech company that contains highly sensitive
+                        data from their customers, SoFi emphasizes on security and appreciated Sourcegraph’s{' '}
+                        <a href="/blog/from-saas-to-on-premises">on-prem solution</a>.
+                    </p>
+                    <h2 className="pt-5 pb-1">Move fast and don’t break things</h2>
+                    <p>
+                        As a financial institution, SoFi needs to avoid downtime--but they also need to continuously
+                        innovate to compete. SoFi runs hundreds of microservices. Their fast growth makes it difficult
+                        to maintain a complete list of the published APIs showing the interdependencies of their
+                        services. A common use case for Sourcegraph is to find which microservice is referenced by
+                        another. This ability safeguards against breaking production with code changes and avoids code
+                        duplications. SoFi’s codebase is updated hundreds of times a day making Sourcegraph’s real time
+                        code investigation features with “go to definition” and “find all references” across all
+                        microservices indispensable for SoFi’s engineers’ daily coding activities, mentoring and stack
```

Comment:
I'm not sure "SoFi's" needs pluralization here.

```suggestion
                        microservices indispensable for SoFi engineers’ daily coding activities, mentoring and stack
```